service/Makefile

```
@@ -26,3 +26,15 @@ go-tools:
 	go install "google.golang.org/protobuf/cmd/protoc-gen-go"
 	#go install "github.com/go-kod/kod/cmd/kod"
 	go install "go.uber.org/mock/mockgen"
+
+certs: cert-ca cert-server
```

checkmake (0.2.2)

[warning] Target "certs" should be declared PHONY.

(phonydeclared)